docs/src/main/asciidoc/flyway.adoc

```diff
@@ -448,3 +448,27 @@ quarkus.openshift.init-task-defaults.wait-for-container.image=my/wait-for-image:
 ----
 
 **Note**: In this context globally means `for all extensions that support init task externalization`.
+
+== Known Issues in Flyway
+
+=== Oracle: Multiple schemas in DevServices
+
+When having multiple schemas in Oracle, you can use the `quarkus.flyway.schemas` property to specify the schemas that Flyway should manage.
+However, because this is executed in the DB as the `quarkus` user, you need to either give DBA privileges to the user that is executing the migration (`quarkus`) or perform the necessary DDLs before the devservice starts. This is done with the `quarkus.datasource.devservices.init-script-path` configuration.
```

Comment:
```suggestion
However, because this is executed in the DB as the `quarkus` user, you need to either give DBA privileges to the user that is executing the migration (`quarkus`) or perform the necessary DDLs before the Dev Service starts. This is done with the `quarkus.datasource.devservices.init-script-path` configuration.
```